docs/ELEKTRA_BACKEND.md

```diff
@@ -0,0 +1,59 @@
+# Elektra backend
+
+## Running a program with the Elektra backend
+
+(We have currently tested this in Arch Linux and OpenSUSE only)
+
+**Prerequisite**
+
+We assume that you already have the Qt5 libraries installed installed.
+Apart from that you will need the following dependencies:
+
+* CMake >= 3.5
+* ECM >= 5.64.0
+* Elektra >= 0.9 ([Installation guides](https://www.libelektra.org/installation/installation))
```

Comment:
Pinpoint it exactly to 0.9.1. It easily can happen we make some braking change so that it will stop working.